### PR TITLE
Added TEST-42852 Get object tags count using get-object

### PR DESCRIPTION
**Pull request by akshaym99**
Signed-off-by: akshaym99 <akshay.s.mankar@seagate.com>

# Problem Statement
- Test Automation for F-28B: Get object tags count using get-object
- TEST-42852

# Design
-  TD doc: https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/983465985/F-28B+S3+Object+Tagging+API

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by akshaym99** [comment]
pytest collection is green: on branch CORTX-31884_tagging_TA
test_id': 'TEST-34217', 'marks': ['security']}, {'nodeid': 'tests/security/test_cortx_port_scanner_netstat.py::test_cortx_port_scanner_netstat', 'test_id': 'TEST-34218', 'marks': ['security', 'skip']}] created at /root/akshay/cortx-test-1/log/te_meta.json
Successfully unmounted directory

================================================================================================ 2296 tests collected in 9.70s =================================================================================================
(virenv) [root@ssc-vm-g4-rhev4-0125 cortx-test-1]#

**Comment by akshaym99** [comment]
@apoorva-rao , @VikasKumarC , @DhobaleVishal please review/approve

File: tests/s3/test_object_tagging.py [hunk 1]
```diff
@@ -1696,3 +1697,32 @@ def test_mpupload_casesensitive_tag_42778(self):
...
+        self.log.info("Uploading an object %s to a bucket %s", self.object_name, self.bucket_name)
+        resp = self.s3_test_obj.put_object(self.bucket_name, self.object_name, self.file_path)
+        assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Object is uploaded to a bucket")
+        self.log.info("Adding tags to an existing object")
+        start_time = perf_counter()
+        resp = self.tag_obj.set_object_tag(self.bucket_name, self.object_name,
+                                           S3_OBJ_TST["s3_object"]["key"],
+                                           S3_OBJ_TST["test_9434"]["value"], tag_count=1)
+        end_time = perf_counter()
```
Comment (DhobaleVishal) [inline tests/s3/test_object_tagging.py:1722]:
```suggestion
        end_time = perf_counter_ns()
```

Comment (DhobaleVishal) [reply tests/s3/test_object_tagging.py:1722]: This gives more accurate time in terms of milliseconds.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

File: tests/s3/test_object_tagging.py (hunk 1 above)
Comment (akshaym99) [reply tests/s3/test_object_tagging.py:1722]: For this particular test case time in seconds is more relatable.

**Review by DhobaleVishal — APPROVED** [review]

**Review by apoorva-rao — APPROVED** [review]

**Review by sonalk0209 — APPROVED** [review]
Approving for merge!!!

**Pull request merged**